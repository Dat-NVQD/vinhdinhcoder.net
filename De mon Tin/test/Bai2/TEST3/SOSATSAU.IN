50
94382105135077691231
41784837088015057622
4502446172