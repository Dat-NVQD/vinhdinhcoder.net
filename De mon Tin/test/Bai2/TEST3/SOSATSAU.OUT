94382105135077691231417848370880150576224502446217
